error during teadorn on skipping tests.

if first test was skipped, we got error in teardown for pytest_responses.
because pytest_runtest_setup is not called on skipping tests but pytest_runtest_teardown is called.

see https://github.com/pytest-dev/pytest/issues/3385

```
___________________________________________________________________________________________ ERROR at teardown of test1 ____________________________________________________________________________________________

item = <Function test1>

    def pytest_runtest_teardown(item):
        if not get_withoutresponses_marker(item):
            try:
>               responses_.stop()

../../.local/share/virtualenvs/pytest_sample-j3KXi7ce/lib/python3.7/site-packages/pytest_responses.py:31:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <responses.RequestsMock object at 0x7fe17f370d30>, allow_assert = True

    def stop(self, allow_assert=True):
>       self._patcher.stop()
E       AttributeError: 'RequestsMock' object has no attribute '_patcher'
```

--- a/test_pytest_responses.py
+++ b/test_pytest_responses.py
@@ -43,3 +43,16 @@ class TestReset:
     def test_2(self):
         responses.add(responses.GET, 'http://responses.invalid', json=2)
         assert requests.get('http://responses.invalid').json() == 2
+
+
+class TestSkip:
+    """
+    Ensure that we don't choke on skipped tests.
+    """
+    @pytest.mark.skip
+    def test_1(self):
+        assert 2 + 2 == 5
+
+    def test_2(self):
+        responses.add(responses.GET, 'http://responses.invalid', json='foo')
+        assert requests.get('http://responses.invalid').json() == 'foo'

